### DISCUSSION TITLE
Option to keep TOC visible

### DISCUSSION THREAD

**Discussion by pbxqdown** [mmistakes/minimal-mistakes#2665] [Q&A] [answered]
## Summary
An option to keep toc visible when browsing long post.

## Motivation
For long articles, toc only displays at the top of page. It would be inconvenient to navigate between sections if toc is not always visible.

**Comment by iBug** [comment, marked as answer]
https://mmistakes.github.io/minimal-mistakes/docs/helpers/#enabled-via-yaml-front-matter

**Comment by pbxqdown** [comment]
Thanks! Sorry i don't why i ignored the section when i read the docs...

**Comment by gricn** [comment]
@iBug Minimal mistakes' Guidance needs to improve.
[Layouts page](https://mmistakes.github.io/minimal-mistakes/docs/layouts/#table-of-contents) doesn't have "**toc_sticky**" parameter. This problem also confused me for a long time.
